### DISCUSSION TITLE
Custom refund schedule with flexible refund rates

### DISCUSSION THREAD

**Discussion by EduNetEurope** [pretix/pretix#4339] [Ideas] [answered]
Hello pretix team, @raphaelm, @pc-coholic,

For non-profits like us, it is a blessing that Pretix is open-source. Thank you! 

Unless I have missed something, it does not seem possible to set a date until when paid-for tickets can be cancelled with a 100% refund, ideally issued automatically. This date should be independent of the presale date.

Ideally, we could set several dates with different refund rates (100% refund when cancelled more than 30 days before the event, 80% refund when cancelled more than 14 days before the event, etc.).

Is such a feature on your roadmap? It would be brilliant.

Thank you for considering it.

**Comment by raphaelm** [comment]
You can set up self-service refunds at Settings>Cancellation.

Multiple dates with different refund rates are currently not supported/planned.

**Reply by EduNetEurope** [reply]
Hello @raphaelm,
Thank you very much for your reply.
How about setting a specific date until when paid-for-tickets can be cancelled?

**Reply by raphaelm** [reply, marked as answer]
It's right there:

![2024-07-29T09:23:23+02:00](https://github.com/user-attachments/assets/557850e3-64e4-459f-bae8-64889c1fe3d2)

**Reply by EduNetEurope** [reply]
Thank you, @raphaelm, I am sorry I missed it.
